frontend/src/hooks/Gitea.js

```diff
@@ -106,12 +72,7 @@ export const useSearchRepos = (
  */
 export const useUserRepos = (username, swrOpts = {}) => {
   const endpoint = `${giteaApiUrl}/users/${username}/repos`
-
-  const { data, error, mutate } = useSWR(
-    username ? endpoint : null,
-    fetcher,
-    swrOpts,
-  )
+  const { data, error, mutate } = useSWR(endpoint, fetcher, swrOpts)
```

Comment:
So this will end up fetching `/users/null/repos` when you use `useAllRepos` no?

Reply:
`useAllRepos` relies on `useSearch` not `useUserRepo`. Actually, `useAllRepos` isn't used anywhere